inventory/staging.yml

```diff
@@ -40,3 +43,10 @@ all:
         aws_s3_custom_domain: muni-portal-backend-staging.s3.amazonaws.com
         media_url: https://muni-portal-backend-staging.s3.amazonaws.com/
         collaborator_api_base_url: https://citizencollab.collaboratoronline.com
+
+    wazimap-ng:
+      hosts:
+        wazimapng1-aws.openup.org.za:
+      vars:
+        app_domain: staging-aws-test.wazimap-ng.openup.org.za
```

Comment:
change to staging.wazimap-ng.openup.org.za before finishing migration and merging